debug_module.tex

```diff
@@ -52,18 +52,20 @@ \section{Debug Module Interface (DMI)} \label{dmi}
 \section{Reset Control} \label{reset}
 
 The Debug Module controls a global reset signal, which can
-reset, or hold in reset, every component in the platform,
+reset, or hold in reset, every component in the platform
 except for the Debug Module and Debug
 Transport Modules. The purpose of this feature is to allow debugging
 programs from the first instruction executed, so exactly what is affected
 by this reset is implementation dependent. The Debug Module's own state and registers are
-generally reset at power-up and if and only if
```

Comment:
How about:
generally reset at power-up and when \Fdmactive in \Rdmcontrol is 0.
Should it be "while" to indicate a level-sensitive signal?

"if and only if" confuses me since it also happens at reset.